code/weapon/weapon.h

```diff
@@ -538,7 +540,6 @@ extern int Num_player_weapon_precedence;				// Number of weapon types in Player_
 extern int Player_weapon_precedence[MAX_WEAPON_TYPES];	// Array of weapon types, precedence list for player weapon selection
 
 #define WEAPON_INDEX(wp)			(int)(wp-Weapons)
-#define WEAPON_INFO_INDEX(wip)		(int)(wip-Weapon_info)
```

Comment:
I think it would be better to keep this (and convert it to a function) so that the weird logic of computing the offset is contained in that function instead of being scattered around in the code.

Reply:
Ok.